youtube/public/app.js

```diff
@@ -0,0 +1,87 @@
+let ws = null;
```

Comment:
This doesn't seem to have a life outside where it is defined as a `const`. Could we get rid of this?

Reply:
The linter should have picked it up, soz.